README.rst

```diff
@@ -0,0 +1,161 @@
+
+Anatomical sex, gender (identity), gender expression and sexuality are distinct concepts that often get confused or conflated. Refer to `the Genderbread Person <https://www.genderbread.org/resource/genderbread-person-v4-0>`_ for a quick summary of the concepts. The website also includes bite-sized explanations of these concepts. Pronouns are a form of gender expression which is not directly predictable from someone's gender (e.g., a woman who uses they/them pronouns) or their appearance. Read more in this `IBM blog post about pronouns <https://www.ibm.com/blogs/think/2020/07/gender-pronouns-how-small-words-make-a-big-difference/>`_ and in this `blog post about pronouns by nonbinary linguist Dr. Kirby Conrod <https://kconrod.medium.com/ask-a-linguist-about-pronouns-8add318fbd67>`_.
+
+"Male" and "female" connectors are needlessly related to anatomical sex (genitalia). It's unclear that this terminology should be in the workplace and we can do better with the alternative "plug" and "socket".
+
+"Master/slave" terminology
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+"Master/slave" terminology in tech alludes to the horrific practice of `slavery <https://www.history.com/topics/black-history/slavery>`_. Especially in the North America, the history of slavery is very recent, very racialized and thus a source of pain to Black people. Many Black developers have asked for the change to be made because they find the term personally offensive and many non-Black developers also find the term inappropriate (see final paragraph of this section). This is not a new thing - people have been against this terminology for decades; see for example `this news article about "master/slave" terminology from 2003 <https://www.cnn.com/2003/TECH/ptech/11/26/master.term.reut/>`_.
+
+You may be surprised to learn that the "master" branch in the version control software git `comes from the "master/slave" concept <https://mail.gnome.org/archives/desktop-devel-list/2019-May/msg00066.html>`_, and not from master copy. In light of this problematic and painful history, it's a good idea to move away from this terminology.
+
+In the context of nodes, we want to differentiate between special nodes that tell other nodes what to do (the "special" node can create more worker nodes, take them down when unused, distribute work between them, etc.).
+
+In the context of databases, on the other hand, we want to differentiate between the main database and several copies of it - "primary/replica" communicates this transparently. Note that Redis has already made the decision to implement "master/replica" terminology so we are constrained in the commands that they provide (REPLICAOF, CLUSTER REPLICAS, etc.).
```

Comment:
Potentially worth broadening this example, since Redis is one of many systems which has made this change. How should we treat the generic system which has made a naming change here? Perhaps wording such as:
* "Note that some systems, such as Redis, have already made the decision to implement alternate terminology..."
* "...in cases where alternate but inclusive terminology has already been selected, we recommend using the given terms rather than primary/replica..."

Thoughts?

Reply:
I like the idea of broadening it! Resolve if you're happy with what I changed it to :)